tests/integration/test_models/test_full_battery_models/test_lithium_ion/test_thermal_models.py

```diff
@@ -100,7 +99,7 @@ def test_consistent_cooling(self):
         def err(a, b):
             return np.max(np.abs(a - b)) / np.max(np.abs(a))
 
-        self.assertGreater(1e-5, err(solutions["SPMe 1+1D"], solutions["SPMe 2+1D"]))
+        assert 1e-5 > err(solutions["SPMe 1+1D"], solutions["SPMe 2+1D"])
```

Comment:
While it is okay here, I think the convention (whether in programming or in language) is to follow that errors connote a sense of negation and are hence measured as entities that are less than others, so this should have been the other way around. Let's make a note of this somewhere and fix all of them at once in another PR.